### PR TITLE
[Windows] SetID

### PR DESCRIPTION
**Pull request by arifBurakDemiray**

### CODE REVIEW

**Review by AliRKat — APPROVED** [review]
LGTM 🚀

**Review by turtledreams — CHANGES_REQUESTED** [review]
neat! we do have tests for windows rite?

**Comment by arifBurakDemiray** [comment]
> neat! we do have tests for windows rite?

Yes we have, I forgot, I will add those @turtledreams

**Review by AliRKat — APPROVED** [review]

**Review by AliRKat — APPROVED** [review]

File: countlyCommon/countlyCommon/CountlyBase.cs [hunk 1]
```diff
@@ -1640,6 +1644,26 @@ public async Task ChangeDeviceId(string newDeviceId, bool serverSideMerge = fals
...
         }
 
+
+        /// <summary>
+        /// Set the device id
+        /// </summary>
+        /// <param name="newDeviceId">New Id that should be used</param>
+        /// <returns></returns>
+        public async Task SetId(string newDeviceId)
+        {
```
Comment (turtledreams) [inline countlyCommon/countlyCommon/CountlyBase.cs:1654]: need logs

**Review by turtledreams — CHANGES_REQUESTED** [review]

**Review by turtledreams — APPROVED** [review]

**Pull request merged**